vmdb/lib/workers/event_catcher_kubernetes.rb

```diff
@@ -0,0 +1,45 @@
+require 'workers/event_catcher'
+
+class EventCatcherKubernetes < EventCatcher
+  def event_monitor_handle
+    require 'Kubernetes/events/kubernetes_event_monitor'
+    @event_monitor_handle ||= KubernetesEventMonitor.new(
+      @ems.api_endpoint,
+      @ems.api_version
+    )
+  end
+
+  def reset_event_monitor_handle
+    @event_monitor_handle = nil
+  end
+
+  def stop_event_monitor
+    @event_monitor_handle.stop unless @event_monitor_handle.nil?
+    rescue StandardError => err
+      $log.warn("#{log_prefix} Event Monitor Stop errored because [#{err.message}]")
+      $log.warn("#{log_prefix} Error details: [#{err.details}]")
```

Comment:
I don't know actually... other catchers are using $log. I guess it's up to the guidelines you defined about where the catcher should be logged.